idaes_examples/tests/test_browse.py

```diff
@@ -1,10 +0,0 @@
-import pytest
```

Comment:
Was this deletion deliberate?

Reply:
This was done to run the tests as the code installation was failing with the PySimpleGUI error.